integration-tests/suites/base.go

```diff
@@ -234,7 +234,7 @@ func (s *IntegrationTestSuiteBase) GetLogLines(containerName string) []string {
 }
 
 func (s *IntegrationTestSuiteBase) launchContainer(name string, args ...string) (string, error) {
-	cmd := []string{common.RuntimeCommand, "run", "-d", "--name", name}
+	cmd := []string{common.RuntimeCommand, "run", "-d", "--name", name, "--privileged"}
```

Comment:
This will launch all containers as privileged, you probably want to add the flag specifically to the `perf-event-open` container via the `args` parameter.

Reply:
Either that or we want a separate `launchPrivilegedContainer` method that adds the flag.

Reply:
Oh, this one is definitely not what I intended to do, probably a left over after some code shuffling.